Explorer/Assets/DCL/EmojiPanel/EmojiPanelController.cs

```diff
@@ -154,6 +154,22 @@ public void Dispose()
             view.EmojiFirstOpen -= ConfigureEmojiSectionSizes;
             view.SectionSelected -= OnSectionSelected;
             emojiSearchController?.Dispose();
+            
+            foreach (EmojiSectionView sectionView in emojiSectionViews)
+            {
+                foreach (Transform emojiButtonTransform in sectionView.EmojiContainer)
+                {
+                    Object.Destroy(emojiButtonTransform.gameObject);
```

Comment:
Use `UnityObjectUtils.SafeDestroy` in both these two destroys. It protects against destroying objects when app quits which usually calls Dispose methods (it messes up things sometimes).